docs/data/material/migration/migration-v5/migration-v5.md

```diff
@@ -54,6 +54,18 @@ This results in a reduction of the `@mui/material` package size by 2.5MB or 25%
 Instead, using ESM-based CDNs such as [esm.sh](https://esm.sh/) is recommended.
 For alternative installation methods, refer to the [CDN documentation](/material-ui/getting-started/installation/#cdn).
 
+### Autocomplete
+
+#### New reason values added to onInputChange
+
+Three new possible values have been added to the `reason` argument in the `onInputChange` callback of the Autocomplete component:
```

Comment:
@DiegoAndai let me know if you think we should expand on this deprecation entry.